addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/ExtensionOpenApi.java

```diff
@@ -166,6 +174,8 @@ public void importOpenApiDefinition(final URI uri) {
      */
     public List<String> importOpenApiDefinition(
```

Comment:
These methods are also called from the API which means that the UI values will be replaced, I don't think that's the expected behaviour.